site/content/en/docs/Getting started/_post_install.md

```diff
@@ -0,0 +1,19 @@
+### Getting to know Kubernetes
```

Comment:
After installation give them a link to the examples page, so they try on minikube !

also consider,  `kubectl cluster-info`